recipe/meta.yaml

```diff
@@ -35,8 +35,8 @@ outputs:
         - python
         - pip
         - freetype {{ freetype }}
-        - numpy 1.23         # [py<311]
-        - numpy {{ numpy }}  # [py>=311]
+        - numpy 2.0         # [py<313]
+        - numpy 2.1  # [py>=313]
```

Comment:
```suggestion
        - numpy {{ numpy }}  # [py>=313]
```

Reply:
That doesn't work. We want to build with numpy >=2 to have compatibility with both numpy 1 and 2.

Reply:
{{ numpy }} resolves to 2.1 for py 3.13 if you have the `ANACONDA_ROCKET_ENABLE_PY313` variable set, though?

Reply:
Ah, yes right. I read too fast and thought the suggestion was to replace both lines.